omnipaxos/src/sequence_paxos/leader.rs

```diff
@@ -63,7 +63,7 @@ where
                 });
             }
         } else {
-            self.state.0 = Role::Follower;
```

Comment:
Not sure if it is stictly necessary to remove this. We had it here before to prevent the stale leader to continue sending new entries if it has been overtaken.

Reply:
With the new election its only the new leader's seq_paxos that triggers handle_leader(). This is because we are being cautious about exactly when to tell paxos we think we are leader (we want a majority of BLEs to see the ballot first, lest the ballot gets established in paxos and lost in BLE). The idea is that the stale leader's paxos will nearly immediately go into follower mode anyways because it will receive the new leader's Prepare message.